src/sdk/util/crypto.ts

```diff
@@ -11,6 +11,20 @@ export function randomBytes(bytes: number): Uint8Array {
     return rb(new Uint8Array(bytes));
 }
 
+export function randomNumber(min: number, max: number): number {
```

Comment:
`max: number` can be much larger than the 4 random bytes
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/MAX_SAFE_INTEGER

the number of bytes should be based on the value of (max-min)